torchvision/datasets/fakedata.py

```diff
@@ -22,8 +22,6 @@ class FakeData(VisionDataset):
     def __init__(self, size=1000, image_size=(3, 224, 224), num_classes=10,
                  transform=None, target_transform=None, random_offset=0):
         super(FakeData, self).__init__(None)
-        self.transform = transform
```

Comment:
It would be even better if you deleted both occurrences and passed  `transform` and `target_transform` to the constructor of the parent class.

https://github.com/pytorch/vision/blob/8d580a1f087d1758cf181269d13ed717dba0c2bd/torchvision/datasets/vision.py#L9

Reply:
Done. Thanks for this suggestion. In `cifar.py`, `caltech.py` etc, I didn't look up all datasets, is it necessary to pass `transform` and `target_transform` to the constructor? If yes, I'm appreciate to do this in this PR.

Reply:
Yes, this should be done in every dataset, since @fmassa implemented it in #872.